modules/terraform-zscc-sg-aws/versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.7.0"
+      version = "~> 4.52.0"
     }
   }
   required_version = ">= 0.13"
```

Comment:
You seem to follow required_version = ">= 0.13.7, < 2.0.0" but this one doesn't confirm to that

Reply:
fixed in commit 13cda2542b0e1ecb28a8e65b6f59cd5309b13dd5